projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_list.scss

```diff
@@ -12,14 +12,31 @@
 /// @property {Map} background [igx-color: 'surface'] - The list background color.
 /// @property {Map} header-background [igx-color: 'surface'] - The list header background color.
 /// @property {Color} header-text-color [igx-color: ('secondary', 500)] - The list header text color.
+///
 /// @property {Map} item-background [igx-color: 'surface'] - The list item background color.
+/// @property {Map} item-background-hover [igx-color: ('grays', 200),  hexrgba: (#fff)] - The list item hover background.
+/// @property {Map} item-background-active [igx-color: ('grays', 200),  hexrgba: (#fff)] - The active list item background color.
+///
 /// @property {Map} item-text-color [igx-color: 'surface', text-contrast: (), rgba: .74] - The list item text color.
-/// @property {Map} item-title-color [igx-color: 'surface', text-contrast: (), rgba: .74] - The list item title color.
-/// @property {Map} item-subtitle-color [igx-color: ('grays', 700)] - The list item subtitle color.
-/// @property {Map} item-background-hover [igx-color: 'surface', text-contrast: (), rgba: .1] - The list item hover background.
-/// @property {Map} item-background-active [igx-color: 'surface' text-contrast: (), rgba: .1] - The active list item background color.
+/// @property {Map} item-text-color-hover [igx-color: 'surface', text-contrast: (), rgba: .74] - The list item text color.
 /// @property {Map} item-text-color-active [igx-color: 'surface', text-contrast: (), rgba: .74] - The active list item text color.
 ///
+/// @property {Map} item-title-color [currentColor] - The list item title color.
+/// @property {Map} item-title-color-hover [currentColor] - The list item title hover color.
+/// @property {Map} item-title-color-active [currentColor] - The list item title active color.
+///
+/// @property {Map} item-subtitle-color [currentColor] - The list item subtitle color.
+/// @property {Map} item-subtitle-color-hover [currentColor] - The list item subtitle hover color.
+/// @property {Map} item-subtitle-color-active [currentColor] - The list item subtitle active color.
+///
+/// @property {Map} item-action-color [currentColor] - The list item action color.
+/// @property {Map} item-action-color-hover [currentColor] - The list item action hover color.
+/// @property {Map} item-action-color-active [currentColor] - The list item action active color.
+///
+/// @property {Map} item-thumbnail-color [currentColor] - The list item thumbnail color.
+/// @property {Map} item-thumbnail-color-hover [currentColor] - The list item thumbnail hover color.
+/// @property {Map} item-thumbnail-color-active [currentColor] - The list item thumbnail active color.
```

Comment:
Check that all types match the initial value. For instance currentColor is not a of type Map.